### ISSUE TITLE
Error when using Portainerssh in the pipeline

### ISSUE THREAD

**Issue by ovarchenko** [devbranch-vadym/portainerssh#46]
When I want to use Portainerssh in the repository pipeline container, I get the error
```
Connecting to a shell ...
panic: inappropriate ioctl for device
goroutine 1 [running]:
github.com/devbranch-vadym/portainerssh/pkg/wsterm.(*WebTerm).setRawtty(0xc00022d3e0, 0x0)
	/home/runner/work/portainerssh/portainerssh/pkg/wsterm/wsterm.go:64 +0xf2
github.com/devbranch-vadym/portainerssh/pkg/wsterm.(*WebTerm).Run(0xc00022d3e0)
	/home/runner/work/portainerssh/portainerssh/pkg/wsterm/wsterm.go:75 +0x58
main.main()
	/home/runner/work/portainerssh/portainerssh/main.go:26 +0x16b
```

The container cannot start tty inside.
https://github.com/devbranch-vadym/portainerssh/blob/master/pkg/wsterm/wsterm.go#L60

If you can fix this problem, please do it.

**Comment by devbranch-vadym** [comment]
Hi @ovarchenko , can you please clarfiy what you mean by pipeline container?

Also are there any steps to reproduce this?

**Comment by ovarchenko** [comment]
I run the bitbucket pipeline, inside the container I install portainerssh and configure access to the portainer server, run `portainerssh container_in_my_portainer_srv -c command`

The pipeline looks like this
```
image: atlassian/default-image:2
pipelines:
  branches:
    master:
      - step:
          script:
            - wget https://github.com/devbranch-vadym/portainerssh/releases/download/v1.4.0/portainerssh-linux-amd64-1.4.0.tar.gz 
            - tar -xzvf portainerssh-linux-amd64-*.tar.gz
            - cp ./portainerssh /usr/local/bin/
            - mkdir ~/.portainerssh/
            - echo 'api_url:' '"https://my-portainer.com/api"' > ~/.portainerssh/config.yml
            - echo 'user:' '"my-user"' >> ~/.portainerssh/config.yml
            - echo 'password:' '"my-password"' >> ~/.portainerssh/config.yml
            - portainerssh stackname_container_1 -c ls
```
In the last step, an error occurs.
If run command `portainerssh stackname_container_1 -c ls` from the computer, it is executed

**Comment by devbranch-vadym** [comment]
It could be caused by TTY missing in Bitbucket environment.
To confirm, try if this helps reproducing an issue:

https://superuser.com/questions/1430872/how-to-simulate-a-shell-without-tty

**Comment by ovarchenko** [comment]
Yes, it helps to reproduce an issue, if run on a computer `true | portainerssh stackname_container_1 -c ls` same mistake

**Comment by devbranch-vadym** [comment]
Fixed in v1.5.0.
